src/lib/polymer-bootstrap.html

```diff
@@ -37,13 +37,28 @@
       var options = {
         prototype: prototype
       };
-      // NOTE: we're specifically supporting older Chrome versions here 
+      // NOTE: we're specifically supporting older Chrome versions here
       // (specifically Chrome 39) that throw when options.extends is undefined.
       if (prototype.extends) {
         options.extends = prototype.extends;
       }
-      Polymer.telemetry._registrate(prototype);
-      document.registerElement(prototype.is, options);
+
+      try {
```

Comment:
I just heard the v8 PM stay himself, "it used to be the case that try/catch was always de-optimized. That is no longer the case!"